src/wirecloud/platform/workspace/mashupTemplateParser.py

```diff
@@ -153,10 +156,10 @@ async def fill_workspace_using_template(db: DBSession, request: Request, user_fu
     user = workspace.creator
 
     if template.get_resource_type() != MACType.mashup:
-        raise TypeError(f'Unssoported resource type {template.get_resource_type()}')
+        raise TypeError(_(f'Unssoported resource type {template.get_resource_type()}'))
```

Comment:
Tienes un typo en `Unssoported`. No se puede realizar una traducción si ha tenido un reemplazo antes, habría que hacer:
```suggestion
        raise TypeError(_(f'Unsupported resource type %(type)s') % {"type": template.get_resource_type()})
```